js/src/utils/_git.ts

```diff
@@ -0,0 +1,109 @@
+import { exec, execSync } from "child_process";
```

Comment:
Do we need to check/handle anything to gracefully handle edge/browser/etc. environments?

Reply:
We shouldn't --the `execGit` and `execSyncGit` functions both will catch errors and return null

Reply:
We can ignore anyone running in browser, since you'll need secrets etc which you can't expose in the browser anyway. I pushed a commit to change to a dynamic import though so it can still be used elsewhere and won't crash